kivy/uix/actionbar.py

```diff
@@ -291,7 +295,7 @@ def _toggle_dropdown(self, *largs):
         children = ddn.container.children
 
         if children:
-            ddn.width = max([self.width, children[0].minimum_width])
+            ddn.width = self.dropdown_width or max([self.width, children[0].minimum_width])
```

Comment:
This always assigns what ever self.dropdown_width, which is the left side of the or statement. But if I don't set that, then my width will always be 0.